jcl/src/jdk.attach/share/classes/com/ibm/tools/attach/attacher/OpenJ9VirtualMachine.java

```diff
@@ -1,6 +1,8 @@
 /*[INCLUDE-IF Sidecar16]*/
 package com.ibm.tools.attach.attacher;
 
+import java.io.ByteArrayInputStream;
+
```

Comment:
Please move this together with the group of import statements below.
Ideally, the copyright comment should precede the package declaration.

Reply:
Good plan.  I took the liberty of reformatting the entire import list.